UWP/HockeyAppDemoUwp/cs/App.xaml.cs

```diff
@@ -128,5 +146,23 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
 
         // Add any application contructor code in here.
         partial void Construct();
+
+        private bool HockeyAppConfigured = false;
+        private void ConfigureHockeyApp()
+        {
+            if (HockeyAppConfigured)
```

Comment:
There is the method CheckForInitialization(), but it throws an exception if it hasn't been configured, and I didn't want to throw an exception every time the app is resumed